earth_enterprise/src/third_party/skia/SConscript

```diff
@@ -100,6 +100,9 @@ sgl_env.UpdateCppflagsForSkia()
 
 cxx_flags = sgl_env['CXXFLAGS']
 cxx_flags.remove('-Werror')
+cxx_flags.append('-fPIC') # Required on some platforms for generating dynamically linked libraries
```

Comment:
Style: If a comment doesn't fit on the same line as the code it's commenting on, we could put the comment above the code line.